extensions/amp-next-page/amp-next-page.md

```diff
@@ -25,237 +25,272 @@ limitations under the License.
 
-If loading the next document is successful it will be appended to the end of
-the current document as a child of the `amp-next-page` component, moving any
-content after it further down the page. Because of this the component should
-usually be placed directly after the unique content of a given page: for
-example, at the end of a news article or recipe, but before the footer or
-other content repeated across articles.
+## Usage
 
-The component will render a total maximum of three documents on screen in a
-component will render a maximum of three documents (total) on screen at one single instance. This limit may be changed or removed in the future.
+The `<amp-next-page>` component loads content pages one after another creating an infinite-scroll experience.
 
-[tip type="important"]
-**Important** [`<amp-analytics>`](../amp-analytics/amp-analytics.md) is [currently unsupported](https://github.com/ampproject/amphtml/issues/15807) on pages users land on through `<amp-next-page>`.
-Tracking page views is supported through [`<amp-pixel>`](../../builtins/amp-pixel.md) or `<amp-analytics>` on the host page.
-[/tip]
+### Configure and load pages
 
-### Analytics triggers
+Specify pages in a JSON configuration. Load the JSON configuration from a remote URL with the `src` attribute, or inline it within a `<script>` child element of `<amp-next-page>`. You may choose to both specify a remote URL and inline the JSON object in order to speed up the initial loading of suggestions. In case the `deep-parsing` attribute is present, `<amp-next-page>` will also recursively read suggestions from each loaded document's `<amp-next-page>` element. The component automatically handles loading and unloading the documents from memory for a smooth user experience.
 
-Partial support for analytics is included through the initial host page via two separate events. These are triggered by `<amp-next-page>` and you can track them in your [amp-analytics](https://amp.dev/documentation/components/amp-analytics) config:
+Documents append to the end of the current document as a child of the `<amp-next-page>` element. To prevent shifting page content down, this component must be the last child of the document `<body>`. If needed, any footer content should be embedded inside the `<amp-next-page>` tag and will be displayed once no more article suggestions are available.
 
-| Event                  | Fired when                                               |
-| ---------------------- | -------------------------------------------------------- |
-| `amp-next-page-scroll` | The user scrolls to a new page                           |
-| `amp-next-page-click`  | The user click on an article from the recommendation box |
+#### Inline Configuration
```

Comment:
I don't think this section is introduced correctly, I'm thinking we need to say something like:

```
Inline a JSON configuration in the <amp-next-page> component by placing it inside a child <script> element. 

<amp-next-page>
  <script type="application/json">
    [
      {
        "url": ...,
        "title": ...,
        "image": ...
      },
      ...
    ]
  </script>
</amp-next-page>
```